run

```diff
@@ -146,6 +146,8 @@ rsync_back() {
 
 # set GOPRIVATE to skip go-proxy on upbound by default - https://tip.golang.org/cmd/go/#hdr-Module_configuration_for_non_public_modules
 GOPRIVATE="github.com/upbound/*"
+# set GOPROXY
+GOPROXY=https://proxy.golang.org
```

Comment:
Go proxy is enabled by default in 1.13 and I don’t think this is default value which would be more like https://proxy.golang.org,direct

Reply:
Reverted this.